com.unity.render-pipelines.universal/Editor/Camera/URPCameraUI.Skin.cs

```diff
@@ -0,0 +1,89 @@
+            public static GUIContent antialiasing = EditorGUIUtility.TrTextContent("Anti-aliasing", "The anti-aliasing method to use.");
+            public static GUIContent antialiasingQuality = EditorGUIUtility.TrTextContent("Quality", "The quality level to use for the selected anti-aliasing method.");
+            public static GUIContent stopNaN = EditorGUIUtility.TrTextContent("Stop NaN", "Automatically replaces NaN/Inf in shaders by a black pixel to avoid breaking some effects. This will affect performances and should only be used if you experience NaN issues that you can't fix. Has no effect on GLES2 platforms.");
+            public static GUIContent dithering = EditorGUIUtility.TrTextContent("Dithering", "Applies 8-bit dithering to the final render to reduce color banding.");
+
+#if ENABLE_VR && ENABLE_XR_MODULE
+            public static GUIContent[] xrTargetEyeOptions =
+            {
+            EditorGUIUtility.TrTextContent("None"),
+            EditorGUIUtility.TrTextContent("Both"),
+        };
+            public static int[] xrTargetEyeValues = { 0, 1 };
+            public static readonly GUIContent xrTargetEye = EditorGUIUtility.TrTextContent("Target Eye", "Allows XR rendering if target eye sets to both eye. Disable XR for this camera otherwise.");
+#endif
+            public static readonly GUIContent targetTextureLabel = EditorGUIUtility.TrTextContent("Output Texture", "The texture to render this camera into, if none then this camera renders to screen.");
+
+            public static readonly string hdrDisabledWarning = "HDR rendering is disabled in the Universal Render Pipeline asset.";
+            public static readonly string mssaDisabledWarning = "Anti-aliasing is disabled in the Universal Render Pipeline asset.";
+
+            public static readonly string missingRendererWarning = "The currently selected Renderer is missing from the Universal Render Pipeline asset.";
+            public static readonly string noRendererError = "There are no valid Renderers available on the Universal Render Pipeline asset.";
+            public static readonly string disabledPostprocessing = "Post Processing is currently disabled on the current Universal Render Pipeline renderer.";
+
+            public static GUIContent[] cameraBackgroundType =
+            {
+            EditorGUIUtility.TrTextContent("Skybox"),
+            EditorGUIUtility.TrTextContent("Solid Color"),
+            EditorGUIUtility.TrTextContent("Uninitialized"),
+        };
+
+            public static int[] cameraBackgroundValues = { 0, 1, 2 };
+
+            // Using the pipeline Settings
+            public static GUIContent[] displayedCameraOptions =
+            {
+            EditorGUIUtility.TrTextContent("Off"),
+            EditorGUIUtility.TrTextContent("Use Pipeline Settings"),
+        };
+
+            public static int[] cameraOptions = { 0, 1 };
+
+            // Camera Types
+            public static GUIContent[] cameraTypeNames { get; } =
```

Comment:
why did we change from:

            public static List<GUIContent> m_CameraTypeNames = null;
            public static readonly string[] cameraTypeNames = Enum.GetNames(typeof(CameraRenderType));
to this?

Reply:
Instead of having a method to transform string to `GUIContent`, I just left the property, directly obtaining the `GUIContent` s from the `string`. I've added a comment explaingin this. Thanks for double checking :)